jest.config.js

```diff
@@ -13,7 +13,6 @@ module.exports = {
   bail: 1,
   verbose: true,
   testPathIgnorePatterns: ['/node_modules/'],
-  testEnvironment: 'node',
```

Comment:
Why this change?

Reply:
This line was duplicated in the config file, present at both line 12 and 16.